src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.cs

```diff
@@ -13,14 +13,14 @@ public abstract partial class Formatter : System.Runtime.Serialization.IFormatte
         protected System.Runtime.Serialization.ObjectIDGenerator m_idGenerator;
         protected System.Collections.Queue m_objectQueue;
         protected Formatter() { }
-        public abstract System.Runtime.Serialization.SerializationBinder Binder { get; set; }
+        public abstract System.Runtime.Serialization.SerializationBinder? Binder { get; set; }
         public abstract System.Runtime.Serialization.StreamingContext Context { get; set; }
-        public abstract System.Runtime.Serialization.ISurrogateSelector SurrogateSelector { get; set; }
+        public abstract System.Runtime.Serialization.ISurrogateSelector? SurrogateSelector { get; set; }
         public abstract object Deserialize(System.IO.Stream serializationStream);
-        protected virtual object GetNext(out long objID) { throw null; }
-        protected virtual long Schedule(object obj) { throw null; }
+        protected virtual object? GetNext(out long objID) { throw null; }
+        protected virtual long Schedule(object? obj) { throw null; }
         public abstract void Serialize(System.IO.Stream serializationStream, object graph);
-        protected abstract void WriteArray(object obj, string name, System.Type memberType);
+        protected abstract void WriteArray(object? obj, string name, System.Type memberType);
```

Comment:
Why is `obj` nullable?

Reply:
I didn't found any implementations for these method. Thought null object still could be serialized, no?

Reply:
https://source.dot.net/#System.Runtime.Serialization.Formatters/System/Runtime/Serialization/Formatter.cs,89

Reply:
yes it will not called with null, I meant overwritten implementation, will update